xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm

```diff
@@ -34,13 +34,23 @@
 </div>
 #if ($editor != 'inline')
 <div id="titleinput" class="form-group">
-  <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
-  <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
+    <dl>
+        <dt>
+          <label for="xwikidoctitleinput">Title</label>
+        </dt>
+        <dd>
+          <input type="text" id="xwikidoctitleinput" name="title" value="Home" class="">
```

Comment:
Please use the same input tag (`<input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>`) and don't change anything.

Reply:
But doesn't $!docTitle return Page's title?

Reply:
For the Title label as in:
```
<label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
```
You are using:
```
<label for="xwikidoctitleinput">Title</label>
```
While it should be the same. You don't need to edit that.

Reply:
It does and it gets the current title of the page and makes it an input value but you don't need to know the details at this point.
Just don't change the input tag. You are removing all the important velocity code.

You just need to change the structure with `<dl>` and `<dd>` tags, not the inputs and labels.